src/Illuminate/Foundation/helpers.php

```diff
@@ -522,41 +522,10 @@ function method_field($method)
      * @param  string  $path
      * @param  string  $manifestDirectory
      * @return \Illuminate\Support\HtmlString
-     *
-     * @throws \Exception
      */
     function mix($path, $manifestDirectory = '')
     {
-        static $manifest;
-
-        if (! starts_with($path, '/')) {
-            $path = "/{$path}";
-        }
-
-        if ($manifestDirectory && ! starts_with($manifestDirectory, '/')) {
-            $manifestDirectory = "/{$manifestDirectory}";
-        }
-
-        if (file_exists(public_path($manifestDirectory.'/hot'))) {
-            return new HtmlString("//localhost:8080{$path}");
-        }
-
-        if (! $manifest) {
-            if (! file_exists($manifestPath = public_path($manifestDirectory.'/mix-manifest.json'))) {
-                throw new Exception('The Mix manifest does not exist.');
-            }
-
-            $manifest = json_decode(file_get_contents($manifestPath), true);
-        }
-
-        if (! array_key_exists($path, $manifest)) {
-            throw new Exception(
-                "Unable to locate Mix file: {$path}. Please check your ".
-                'webpack.mix.js output paths and try again.'
-            );
-        }
-
-        return new HtmlString($manifestDirectory.$manifest[$path]);
+        return app('mix')->mix($path, $manifestDirectory);
```

Comment:
Yeah I thought about that.
`create` isn't a good one either because you don't _create_ anything. I didn't find something I liked, so I kept that because in any way, the method will always be called by the `mix()` helper, as usual.